src/pylutron_caseta/smartbridge.py

```diff
@@ -811,6 +849,18 @@ def _handle_unsolicited(self, response: Response):
         ):
             self._handle_button_led_status(response)
 
+    def _handle_smart_away_status(self, response: Response):
+        _LOG.debug("Handling Smart Away status: %s", response)
+        if response.Body is None:
+            return
+
+        status = response.Body["AwayStatus"]
+        away_id = id_from_href(status["href"])
+        self.smart_away_state = status["EnabledState"]
+        # Notify any subscribers of the change to Smart Away status
+        if "0" in self._smart_away_subscribers:
+            self._smart_away_subscribers["0"](smart_away_state)
```

Comment:
If there's only one ID for this, the subscriber shouldn't be stored in a dictionary of subscribers by ID.